### PR TITLE
Implement additional comments page for skills#1227

### PR DESCRIPTION
**Pull request by fragm3**
Solves first part of #1227

Changes: Added separate component for feedback section. 
Surge Deployment Link: https://pr-1227-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

File: src/components/SkillFeedbackPage/SkillFeedbackPage.js [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+        dataType: 'jsonp',
+        jsonp: 'callback',
+        crossDomain: true,
+        success: function(data) {
+          self.updateData(data.skill_metadata);
+        },
+      });
+      if (cookies.get('loggedIn')) {
+        $.ajax({
+          url: userSkillRatingUrl,
```
Comment (akshatnitd) [inline src/components/SkillFeedbackPage/SkillFeedbackPage.js]: This AJAX call is not required.

File: src/components/SkillFeedbackPage/SkillFeedbackPage.js [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+          error: function(e) {
+            console.log(e);
+          },
+        });
+      }
+      this.getFeedback();
+    }
+  }
+
+  saveRatingOverTime = (ratings_over_time = []) => {
```
Comment (akshatnitd) [inline src/components/SkillFeedbackPage/SkillFeedbackPage.js]: Remove this function

File: src/components/SkillFeedbackPage/SkillFeedbackPage.js [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+            authorUrl={this.state.author_url}
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+SkillFeedbackPage.propTypes = {
+  location: PropTypes.object,
```
Comment (akshatnitd) [inline src/components/SkillFeedbackPage/SkillFeedbackPage.js]: Not required.

Comment (fragm3) [reply src/components/SkillFeedbackPage/SkillFeedbackPage.js]: I think this is required, without this it was giving: 'location' is missing in props validation.

**Comment by fragm3** [comment]
@akshatnitd made the changes, please review

File: src/components/SkillFeedbackPage/SkillFeedbackPage.js [hunk 4]
```diff
@@ -0,0 +1,575 @@
...
+      this.setState({ showAuthorSkills: true });
+    }
+  };
+
+  closeAuthorSkills = () => {
+    this.setState({ showAuthorSkills: false });
+  };
+
+  render() {
+    console.log(this.state);
```
Comment (anshumanv) [inline src/components/SkillFeedbackPage/SkillFeedbackPage.js]: Remove

File: src/components/SkillFeedbackPage/SkillFeedbackPage.js [hunk 5]
```diff
@@ -0,0 +1,575 @@
...
+      openDeleteDialog: false,
+      errorText: '',
+      skill_feedback: [],
+      imgUrl: null,
+      dataReceived: false,
+      showAuthorSkills: false,
+    };
+    let clickedSkill = this.props.location.pathname.split('/')[2];
+    this.name = clickedSkill;
+    this.url = urls.API_URL + '/cms/getSkillList.json?group=Knowledge';
```
Comment (anshumanv) [inline src/components/SkillFeedbackPage/SkillFeedbackPage.js]: Is it used anywhere?

**Comment by fragm3** [comment]
@anshuman the author skill component is not rendering, see for skill listing page. I am just using the AuthorSkills component here.

File: src/components/SkillFeedbackPage/SkillFeedbackPage.js (hunk 5 above)
Comment (fragm3) [reply src/components/SkillFeedbackPage/SkillFeedbackPage.js]: Yes, it is used

Comment (anshumanv) [reply src/components/SkillFeedbackPage/SkillFeedbackPage.js]: Ah okay then, I don't see the need to fetch all skills data in the component though or is it something that I might've missed?

Comment (fragm3) [reply src/components/SkillFeedbackPage/SkillFeedbackPage.js]: No, you are correct, I have updated the code. Thanks!

File: src/components/SkillFeedbackPage/SkillFeedbackPage.css [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    text-decoration: underline;
+}
+
+.feedback-name{
+    font-weight: 400;
+}
+
+.feedback-name:hover{
+    text-decoration: underline;
+}
```
[the remark this answers is not in the document]
Comment (fragm3) [reply src/components/SkillFeedbackPage/SkillFeedbackPage.css]: Resolved!

**Comment by fragm3** [comment]
I am working on the pagination part, which changes the whole structure of the comment page. No need to review this, Ill create a new PR.

**Comment by fragm3** [comment]
Closing this, new PR created with pagination

**Pull request closed without merging**